src/i18n/es.json

```diff
@@ -45,6 +45,9 @@
     "you_want_create_account": "¿Aún no tienes una cuenta?"
   },
   "reports": {
+    "allow_enable_trial_HTML": "<p>Envía Emails automáticos y 100% personalizados a quienes visiten una página específica de tu sitio web o tienda online. Por ejemplo, puedes enviar un descuento a los usuarios que visitaron la página de un producto en particular para hacer que regresen a comprarlo.</p><p><strong>¿Cuándo finaliza el periodo de prueba?</strong><br />Nuestro objetivo es que puedas ver de qué forma utilizar esta funcionalidad en tu negocio y comprobar sus beneficios, por lo que por el momento podrás utilizarla sin limitaciones.</p><p><strong>¿Qué sucederá con las Automations creadas cuando finalice el periodo de prueba?</strong><br />Permanecerán inactivas, y no podrás editarlas hasta que contrates el módulo especial que incluye este servicio.</p>",
```

Comment:
What do you think about using headers (h1, h2, h3, etc) in place of strong and brs?